ocaml/idl/datamodel_host.ml

```diff
@@ -2338,6 +2347,15 @@ let emergency_clear_mandatory_guidance =
     ~doc:"Clear the pending mandatory guidance on this host"
     ~allowed_roles:_R_LOCAL_ROOT_ONLY ()
 
+let configure_ssh =
+  call ~name:"configure_ssh" ~doc:"Configure sshd service" ~lifecycle:[]
+    ~params:
+      [
+        (Ref _host, "self", "The host")
+      ; (ssh_status, "status", "Status of sshd service")
```

Comment:
I would recommend to have 2 calls, one 2 turn it on, and one to turn it off. This is because enums tend to be problematic for upgrades, so I'd rather avoid them